### ISSUE TITLE
Operand data type bit is invalid for max operator

### ISSUE THREAD

**Issue by mariojsnunes** [sebastienros/yessql#472]
Getting "Operand data type bit is invalid for max operator" error when using this on a query:
`x.With<BooleanFieldIndex>(field => field.ContentField == nameof(BaseContentPart.Property)).OrderByDescending(x => x.Boolean)`
Started working again after I commented it.

Is there any change in the past 2-3 months could cause this? I updated OC to the latest preview today.

**Comment by sebastienros** [comment]
Yes, we are using GROUP BY now to eliminate duplicate documents. I will add a similar unit test.

**Comment by mariojsnunes** [comment]
Hi there. Is there any plans to support ordering boolean fields again?

**Comment by sebastienros** [comment]
Can you give it a try and attempt to fix it?

**Comment by mariojsnunes** [comment]
@sebastienros Sure. I've made a PR #503 to try to replicate the issue. Can you allow for tests to run?

**Comment by jtkech** [comment]
Done
